src/template.js

```diff
@@ -1,43 +1,19 @@
 import React from "react";
 import PropTypes from "prop-types";
-import { withRouter, Link } from "react-static";
-import LOGO from "./static/logo.svg";
-import { Header, Footer } from "formidable-landers";
+import { Footer } from "./components/footer";
+import { GlobalStyle } from "./global-style";
+import { Header } from "./components/header";
+import { withRouter } from "react-static";
+// import { Header, Footer } from "formidable-landers";
```

Comment:
comment left in here

Reply:
or was this on purpose?

Reply:
I left it in as a temporary reminded that these should eventually be replaced with updated remote components if we keep the same system in place. It should definitely be removed though before merging, thanks!

Reply:
I had that thought right after I left the review comment - sounds good!